src/js/visual/configurator-csr.js

```diff
@@ -1771,10 +1779,12 @@ ripe.ConfiguratorCsr.prototype._registerHandlers = function() {
  * @ignore
  */
 ripe.ConfiguratorCsr.prototype._registerInitialsHandlers = function() {
-    this.owner.bind("initials", (initials, engraving, params) =>
+    // unregister previous config handlers to avoid duplicate handlers
+    this._unregisterInitialsHandlers();

```

Comment:
Don't call the unregister method here. If you need to do it here, it's a symptom of a bug

Reply:
The bug cause is `_onPostConfig ` being called several times. This results in binding several initials handlers. Later those initials handlers are not removed because we're only saving the callback of 1 of them.

Reply:
But it being called multiple times is maybe a symptom of some bug of the caller as this is just used to register. If you need to unregister before registering again, then the caller of `_registerInitialsHandlers()` should call `_unregisterInitialsHandlers()`, It's not `_registerInitialsHandlers()` responsibility to call `_unregisterInitialsHandlers()`